### DISCUSSION TITLE
Why not use the "cmfs" parameter for the "XYZ" conversion in the "spds_CIE_*_chromaticity_diagram_plot" functions?

### DISCUSSION THREAD

**Discussion by henczati** [colour-science/colour#698] [Q&A] [answered]
In `colour/plotting/diagrams.py`, `spds_CIE_*_chromaticity_diagram_plot` functions.

**Comment by thomasmansencal** [comment, marked as answer]
I'm somehow right now enforcing _CIE 1931 2 Degree Standard Observer_ usage because there are still fundamental questions I don't have any answers to, the main ones being: The RGB colourspaces we are using (_sRGB_, _ACES2065-1_, _Rec. 2020_) have been defined using the _CIE 1931 2 Degree Standard Observer_ but what happens to their primaries / whitepoint when you use a different observer? Should you update those primaries? How do you do that?

This is related to #25.
